docs/extras/nandrestore.md

```diff
@@ -1,36 +1,39 @@
-## Restore a nand backup to the switch
+## Restore a NAND backup to the switch
 
-!!! danger "Warning:"	
-	-This will reset all of your saves, games, system version and other system settings back to a point when you made the the nand backup. Keep this in mind, as you probably don't have to restore a nand backup unless you have bricked your switch or want to go back online safely after using cfw.
+!!! Danger "Warning:" 	
+	-This will reset your system version, all of your saves, games, and other system settings back to a point from when you made the the NAND backup. Keep this in mind, as you probably don't have to restore a NAND backup unless you have bricked your switch or want to go back online safely after using cfw.
```

Comment:
Why move system version to the first bullet?

Reply:
reverted back to orginal.